backend/services/syson-services/src/main/resources/DirectEdit.g4

```diff
@@ -43,7 +43,6 @@ multiplicityExpressionMember :
 featureExpressions :
 	(subsettingExpression|redefinitionExpression)? (typingExpression)? (valueExpression)?
 	| (typingExpression)? (subsettingExpression|redefinitionExpression)? (valueExpression)?
-	| (transitionExpression)?
```

Comment:
Are you sure about that? With this deletion, how is it possible to edit `transitions`?

Reply:
Furthermore `transitionExpression` is only called here so the rule declared on l.64 is not used anymore.

Reply:
It is possible to have multiple "entry rules" in Antlr. We already use this feature to parse regular direct edit input
```
ParseTree tree = parser.expression();
```
And transition direct edit input
```
ParseTree tree = parser.transitionExpression();
```

Removing `transitionExpression` is thus not a problem, as long as transition-related direct edit services call the appropriate parser.